### ISSUE TITLE
Unable to open project(or anything from the menu) after February 2019 (version 1.32) update on Ubuntu 18.04.1 LTS

### ISSUE THREAD

**Issue by abhinav1602** [Microsoft/vscode#71263]
![Screenshot from 2019-03-27 12-54-29](https://user-images.githubusercontent.com/20699512/55057538-7ed5d880-508f-11e9-94e6-f3765ef4acdc.png)

- VSCode Version:  1.32(February 2019 update)
- OS Version: Ubuntu 18.04.1 LTS

Steps to Reproduce:

1. Install [Vesion 1.32](https://code.visualstudio.com/updates/v1_32#_editor) update after downloading from https://code.visualstudio.com/
2. Try opening a project on VS Code.

Does this issue occur when all extensions are disabled?: Didn't Check before fixing it.

**Comment by weinand** [comment]
What do you mean by "Install updates from https://code.visualstudio.com/updates/v1_32#_editor"?

**Comment by abhinav1602** [comment]
Here is how I got it fixed:
1. Tried reinstalling the earlier update to fix the problem and got error saying:
`E: dpkg was interrupted, you must manually run 'sudo dpkg --configure -a' to correct the problem. ` 
2. `sudo dpkg --configure -a`.
3. Reinstalled the latest update([Vesion 1.32](https://code.visualstudio.com/updates/v1_32#_editor)) again and everything started working normally after that.

P.S: Did not get enough time to figure out why it happened.
